src/movement_controls.py

```diff
@@ -157,14 +166,19 @@ def move(self, delta_time: float, vehicle: arcade.Sprite, walls: arcade.SpriteLi
             self.shadow_sprite.center_y = vehicle.center_y
         else:
             # no wall collisions means a valid spot to move to
-            vehicle.center_x = self.shadow_sprite.center_x
-            vehicle.center_y = self.shadow_sprite.center_y
+            vehicle.location = (
+                self.shadow_sprite.center_x,
+                self.shadow_sprite.center_y,
+                self.shadow_sprite.radians,
+            )
 
-            vehicle.angle = self.shadow_sprite.angle
         # NOTE: place holder boundaries to wrap aroung like pacman
         if self.debug_world_boundary != 0:
-            vehicle.center_x = vehicle.center_x % self.debug_world_boundary
-            vehicle.center_y = vehicle.center_y % self.debug_world_boundary
+            vehicle.location = (
+                vehicle.center_x % self.debug_world_boundary,
+                vehicle.center_y % self.debug_world_boundary,
+                self.shadow_sprite.radians,
```

Comment:
```suggestion
                vehicle.radians,
```
The pac-man-wrap shouldn't change the vehicle's rotation, since that should only happen above where we prove that no wall collisions have happened.  I was hitting a bug where if I turned while hitting a wall, the car's hood would continue to rotate into the wall and get fully stuck.